org-gen/README.md

```diff
@@ -0,0 +1,15 @@
+# Istio Organization
+
+The folder contains configuration for the Istio Github Organization. Data here is automatically synced with Github using [peribolos](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/peribolos).
+
+## Making Changes
+
+To modify the org, simply change the config file and submit a PR. Once the PR is merged, the org will be updated.
+
+Changes can be tested with `make test`.
+
+To add yourself to the Org, please add your name under `members.yaml`. 
+
+To add yourself as a developer, which is given additional access to repos, add yourself under `developers.yaml` instead.
```

Comment:
```suggestion
To add yourself as a developer, which is given additional access to repos, add yourself under `developers.yaml` instead.  You may request to add yourself to the Org and as a developer in one PR, as long as a link to at least 1 merged PR to the Istio Org is provided.
```

Reply:
actually I may be understand this wrong.  looking at latest change, a user either in developer.yaml or members.yaml.

Reply:
it would be good to make some guidance on why people should choose developers.yaml over members.yaml.  Would it be something like if you expect you to continue to contribute to istio org regularly, we recommend you to add yourself to developers.yaml??

Reply:
To be honest, I am not really sure :slightly_smiling_face:. Open to suggestions here... right now I was just trying to generate the current org structure we have today with a simpler yaml configuration

Reply:
must someone have been a member to become a developer?  can you do it in one go?

(I find the terminology confusing. there's nothing about 'developer' in https://github.com/istio/community/blob/master/ROLES.md.  I would assume all members are developers?)

Reply:
@craigbox I am maintaining status quo here in this PR really. I am not sure there is actually value in being a member but not a developer. We can likely just have one set of people - I am open to whatever steering suggests. The currently implementation makes no changes to anyone's membership. To keep things simpler it may make sense to merge this decouple the changes, since right now its trivial to test this PR by diffing the before and after?

Reply:
I'd never heard of 'developer' as a role but it's not my place to know.  I'll defer to @linsun

Reply:
Developer currently has a bit more privilege i believe, but it is something any member can ask for it.  I'm personally in favor of making member=developer but in the past people have shared concerns that they want to limit the folks who can be developers out of all members as the priviledges come with developers to run tests.